src/Entities/Page/PageBuilder.cs

```diff
@@ -544,36 +544,37 @@ public PageBuilder WithAttachmentsFactory(Func<ValueTask<IEnumerable<FileAttachm
     /// <inheritdoc/>
     IPage IPageBuilder<IPage>.Build() => Build();
 
-    internal PageBuilder WithPaginatorFooter(PaginatorFooter footer, int page, int totalPages, ICollection<IUser>? users)
+    internal PageBuilder WithPaginatorFooter(PaginatorFooter footer, int currentPageIndex, int maxPageIndex, ICollection<IUser>? users)
     {
         if (footer == PaginatorFooter.None)
-        {
             return this;
-        }
 
         Footer = new EmbedFooterBuilder();
+
         if (footer.HasFlag(PaginatorFooter.Users))
         {
-            if (users is null || users.Count == 0)
+            if (users?.Any() != true)
```

Comment:
`Count` should be preferred over `Any`, it's more clear too.

```suggestion
            if (users is null || users.Count == 0)
```